docs/workflows/cosign-support.rst

```diff
@@ -0,0 +1,102 @@
+.. _cosign-support:
+
+Mirror cosign signatures
+========================
+
+Being an OCI compliant registry Pulp Container registry can natively mirror cosign signatures
+wich are stored as an OCI image::
+
+ {
+    "schemaVersion": 2,
+    "mediaType": "application/vnd.oci.image.manifest.v1+json",
+    "config": {
+        "mediaType": "application/vnd.oci.image.config.v1+json",
+        "digest": "sha256:f35028aa1563f37ccbaa0b32c57777ffbd8e9e3d81d739fec0022995e58a375a",
+        "size": 153
+    },
+    "layers": [
+        {
+            "mediaType": "application/vnd.dev.cosign.simplesigning.v1+json",
+            "digest": "sha256:d3370bd32b32aba43de2b45bb4a2de2fb5c95fd2edbe738acbc3bc595b80c456",
+            "size": 305,
+            "annotations": {
+                "dev.cosignproject.cosign/signature": "MEUCIBWDnTKhbf5x3mSuEHWkv3ixloIFXeDpfXipF9szqrd5AiEA+UU5J84gQ9JnmT6QZAXiPXqSoDVW0CXQYssGh63e9Ro="
+            }
+        }
+    ]
+ }
+
+
+During the syncronization task, Pulp will automatically mirror cosign signatures or atomic
+signatures (accessible via signatures extentions API).
+
+
+Sign and push cosign signatures
+===============================
+
+Pulp Container registry can host cosign signature which can be pushed via cosign or podman clients:
```

Comment:
```suggestion
The Pulp Registry can host cosign signatures which can be pushed via cosign or podman clients:
```

Reply:
I prefer saying pulp container registry. there might be other pulp registries of a different nature, not container.